sdk/keyvault/test-resources.json

```diff
@@ -61,16 +61,7 @@
                 "verify",
                 "wrapKey"
               ],
-              "secrets": [
-                "backup",
-                "delete",
-                "get",
-                "list",
-                "purge",
-                "recover",
-                "restore",
-                "set"
-              ],
+              "secrets": ["backup", "delete", "get", "list", "purge", "recover", "restore", "set"],
```

Comment:
Nit: I'd keep these on separate lines to make diff'ing / reviewing easier.

Reply:
Gotcha. Though, this was automatically formatted by the commit hooks, I believe.